### ISSUE TITLE
Unpin xorg-libice?

### ISSUE THREAD

**Issue by pauldmccarthy** [conda-forge/xorg-libxt-feedstock#14]
### Comment:

Hi @pkgw,

I have found myself in a situation whereby I am unable to install a set of packages from conda-forge due to a conflict regarding `xorg-libice` - some packages have been built against `xorg-libice 1.1`, but this cannot be installed alongside `xorg-libxt`, because this recipe currently pins `xorg-libice` at `1.0`.

Given that you recently [added a `run_exports` section](https://github.com/conda-forge/xorg-libice-feedstock/pull/12) to the `xorg-libice` feedstock, would it make sense to unpin `xorg-libice` in this recipe, and publish a version buitl against `xorg-libice 1.1`?

Thanks!

--- 

I am encountering the conflict when trying to create an environment with the latest versions of VTK and QT.
I have simplified the environment as best as I can down to the following command, which triggers the conflict. 

 * The latest `qt-main` package is built against `xorg-libice 1.1` and `alsa-lib=1.2.9`, so I am adding the latter to force the latest `qt-main` to be selected. 
 * The latest `vtk` package is built against `xorg-libxt`, which in turn is pinning `xorg-libice` at 1.0

```
mamba create -c conda-forge -p ./test.env "qt-main"   "vtk=9.2.*" "alsa-lib=1.2.9"
```

<details>

```
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.4.1) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Looking for: ['qt-main', 'vtk=9.2', 'alsa-lib=1.2.9']

conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
Could not solve for environment specs
The following packages are incompatible
├─ alsa-lib 1.2.9**  is requested and can be installed;
├─ qt-main   is installable with the potential options
│  ├─ qt-main 5.15.8 would require
│  │  ├─ libjpeg-turbo >=2.1.5.1,<3.0a0 , which requires
│  │  │  └─ jpeg <0.0.0a , which can be installed;
│  │  ├─ libxml2 >=2.11.4,<2.12.0a0 , which can be installed;
│  │  └─ xorg-libice >=1.1.1,<2.0a0 , which can be installed;
│  ├─ qt-main [5.15.2|5.15.3] would require
│  │  └─ alsa-lib >=1.2.3,<1.2.4.0a0 , which conflicts with any installable versions previously reported;
│  ├─ qt-main [5.15.3|5.15.4] would require
│  │  └─ alsa-lib >=1.2.3.2,<1.2.4.0a0 , which conflicts with any installable versions previously reported;
│  ├─ qt-main 5.15.4 would require
│  │  └─ alsa-lib >=1.2.6.1,<1.2.7.0a0 , which conflicts with any installable versions previously reported;
│  ├─ qt-main 5.15.6 would require
│  │  └─ alsa-lib >=1.2.7.2,<1.2.8.0a0 , which conflicts with any installable versions previously reported;
│  └─ qt-main [5.15.6|5.15.8] would require
│     └─ alsa-lib >=1.2.8,<1.2.9.0a0 , which conflicts with any installable versions previously reported;
└─ vtk 9.2**  is uninstallable because there are no viable options
   ├─ vtk [9.2.2|9.2.5|9.2.6] would require
   │  └─ jpeg >=9e,<10a , which conflicts with any installable versions previously reported;
   ├─ vtk 9.2.6 would require
   │  └─ libxml2 >=2.10.3,<2.11.0a0 , which conflicts with any installable versions previously reported;
   ├─ vtk 9.2.6 would require
   │  └─ vtk-base [9.2.6 egl_py310h1234567_3|9.2.6 egl_py310h1234567_4|...|9.2.6 qt_py39h1234567_205], which requires
   │     └─ libxml2 >=2.10.4,<2.11.0a0 , which conflicts with any installable versions previously reported;
   └─ vtk 9.2.6 would require
      └─ vtk-base [9.2.6 egl_py310h1234567_6|9.2.6 egl_py310h1234567_7|...|9.2.6 qt_py39h1234567_208], which requires
         └─ xorg-libxt [ |>=1.3.0,<2.0a0 ], which requires
            └─ xorg-libice 1.0.* , which conflicts with any installable versions previously reported.
```

</details>

**Comment by larsoner** [comment]
Agreed, I think this is likely the cause of https://github.com/conda-forge/vtk-feedstock/pull/302 / the inability to install `vtk` alongside latest `qt-main`, `tensorflow`, etc. (because `vtk` uses `xorg-libxt` and these other packages use `libice >= 1.1`)

**Comment by larsoner** [comment]
... or to make a more minimal example, `xorg-libxt` won't be compatible with any newer builds of `qt-main`:
```
$ mamba create --override-channels -c conda-forge -n test --dry-run "python=3.11.4" "qt-main ==5.15.8=*_15" "xorg-libxt ==1.3.0=hd590300_0"
...
Looking for: ['python=3.11.4', 'qt-main==5.15.8[build=*_15]', 'xorg-libxt==1.3.0=hd590300_0']

conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
Could not solve for environment specs
The following packages are incompatible
├─ qt-main 5.15.8 *_15 is installable and it requires
│  └─ xorg-libice >=1.1.1,<2.0a0 , which can be installed;
└─ xorg-libxt ==1.3.0 hd590300_0 is uninstallable because it requires
   └─ xorg-libice 1.0.* , which conflicts with any installable versions previously reported.
```

**Comment by pauldmccarthy** [comment]
Should be fixed by #15, thanks @Tobias-Fischer @hmaarrfk !
